app/components/addItemForm.tsx

```diff
@@ -0,0 +1,98 @@
+"use client";
+import React from "react";
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import Grid from "@mui/material/Grid";
+import MenuItem from "@mui/material/MenuItem";
+import Select from "@mui/material/Select";
+import Paper from "@mui/material/Paper";
+import Box from "@mui/material/Box";
+import { ShopItem, ShopItems } from "@/lib/features/items/itemsSlice";
+import { useAppDispatch } from "@/lib/hooks";
+import { useForm, SubmitHandler } from "react-hook-form";
+import { TextField } from "@mui/material";
+import { updateItem } from "@/utils/client";
+
+interface Props {
+  item: ShopItem;
+  itemKey: number;
+}
+
+export default function AddItemForm() {
```

Comment:
```suggestion
import { FC } from 'react';
const AddItemForm: FC = () => {
...
}
export default AddItemForm;
```
FC - functional component, where you can add props in generic